README.md

```diff
@@ -1,5 +1,13 @@
 # Graasp Apps Query Client
 
+[![npm (scoped)](https://img.shields.io/npm/v/@graasp/apps-query-client?logo=npm)](https://www.npmjs.com/package/@graasp/apps-query-client)
+[![GitHub package.json version](https://img.shields.io/github/package-json/v/graasp/graasp-apps-query-client?color=deepskyblue&logo=github)](https://github.com/graasp/graasp-apps-query-client/releases/latest)
+![npm](https://img.shields.io/npm/dm/@graasp/apps-query-client?color=green)
```

Comment:
What's the difference between this and `npm (scoped)` ? I see from the badge that it is for downloads, maybe we could write it into the alternative text?

Reply:
Yes good point !